packages/@okta/vuepress-theme-prose/components/GuidesOverview.vue

```diff
@@ -1,25 +1,47 @@
 <template>
-  <div class="guides-overview">
-    <GuidesNavigation />
-    <div class="overview-content">
-      <h1>Guides</h1>
+  <div class="content--container">
+    <div class="tree-nav">
+      <Sidebar :sidebarActive="treeNavOpen"/>
+    </div>
+    <div class="content-area">
+      <h1>{{ $page.title }}</h1>
       <Content />
       <h2>Featured Guides</h2>
-      <div class="featured">
-         <FeaturedGuide v-for="guideName in featured" :guideName="guideName"/>
+      <div class="guides--features">
+         <FeaturedGuide :key="guideName" v-for="guideName in featured" :guideName="guideName"/>
       </div>
```

Comment:
Looks like we're missing the "On This Page" content nav here.

Does the Guides Overview article need to use a separate Vue component? From a UX perspective, the section overview pages are just articles and should use the prose docs layout. Maintaining the same layout in multiple components seems disadvantageous.

Reply:
Having separate component gives us the flexibility of having a different layout for overview page similar to how https://developer.okta.com/docs/reference/ page is currently. It just happens that current GuidesOverview looks similar to prose content page layout.

Reply:
I would say such flexibility is not currently desired. Consistency is more important.

For example, the Reference Overview page should _not_ be using a different layout. All the section overview pages should be using the prose layout, and stay exactly in sync with any future changes to the layout.

Just sounds like more engineering effort if layout changes have be to duplicated several times.

Reply:
oh k. That makes sense. I've removed the GuidesOverview component and now using the common prose layout for overview page. Check out the latest commit!